starters/solidjs-tailwind/package.json

```diff
@@ -2,18 +2,8 @@
   "name": "solidjs-tailwind",
   "version": "0.1.0",
   "private": true,
+  "hasShowcase": true,
   "description": "SolidJS and TailwindCSS",
-  "contributors": [
-    "Maarten Bicknese <maarten.bicknese@thisdot.co>",
-    "Chinedu Okpala <chinedu.okpala@thisdot.co>",
-    "Daian Scuarissi <daian@thisdot.co>",
-    "Mattia Magi <mattia.magi@thisdot.co>",
-    "Mark Shenouda <mark@thisdot.co>",
-    "Victor Umeh <victor.umeh@thisdot.co>",
-    "Dane Grant <dane@thisdot.co>",
-    "Jan Kaiser <jan.kaiser@thisdot.co>",
-    "Jerry Hogan <jerry.hogan@thisdot.co>"
-  ],
```

Comment:
are we sure we want to remove this?
I vaguely remember a discussion a while back about adding contributors in Slack.
So I would check on this

Reply:
oh true!  I deleted this accidentally. Added myself muahahah